src/functionaltest/java/com/ericsson/ei/frontend/TestSwitchBackend.java

```diff
@@ -73,6 +74,16 @@ public void testSwitchBackend() throws Exception {
         switchBackendPage.removeInstanceNumber(1);
         assertEquals("switchBackendPage.presenceOfInstance returned true when it should have been false",
             false, switchBackendPage.presenceOfInstance(1));
+
+        tearDownMocks();
+    }
+
+    private void tearDownMocks() {
```

Comment:
Small detail, tearDownMocks should be under setupMocks.

Is it not possible to run those functiona from a before and after class? That could make it easier to edit the test in future.

Reply:
Done